Documentation update for SCP-B2B-1.0.19 and SCP-4.8.18 (#829)
commerce-docs/microsite-commerce-storefront: src/content/docs/setup/configuration/storefront-compatibility/v248.mdx

- [`customerSegments`](https://developer.adobe.com/commerce/webapi/graphql/schema/customer/queries/customer-segments/) query - Retrieves information about specified customer segment.

> - [`deleteCustomerAddressV2`](https://developer.adobe.com/commerce/webapi/graphql/schema/customer/mutations/delete-address-v2) mutation - Deletes a billing or shipping address of a customer using customer UID as input.

Reviewer: ```suggestion
- [`deleteCustomerAddressV2`](https://developer.adobe.com/commerce/webapi/graphql/schema/customer/mutations/delete-address-v2) mutation - Deletes a customer’s billing or shipping address based on the provided customer UID.
```

Author: Fixed

- [`exchangeExternalCustomerToken`](https://developer.adobe.com/commerce/webapi/graphql/schema/customer/mutations/exchange-external-customer-token/) mutation - Authenticates users via an integration token and returns both the customer token and associated customer data.

> - [`updateCustomerAddressV2`](https://developer.adobe.com/commerce/webapi/graphql/schema/customer/mutations/update-address-v2) mutation - Updates a billing or shipping address of a customer using customer UID as input.

Reviewer: ```suggestion
- [`updateCustomerAddressV2`](https://developer.adobe.com/commerce/webapi/graphql/schema/customer/mutations/update-address-v2) mutation - Updates a customer’s billing or shipping address based on the provided customer UID.
```

Author: Fixed
